### DISCUSSION TITLE
Light transmission of the diffuser screen

### DISCUSSION THREAD

**Discussion by HostoPhoto** [srozum/film_camera_tester#5] [General]
I was wondering if you have the specs on the "Silky white" diffuser screen. A 6mm thickness is not standard i guess, i find 3, 5 or 10mm easy.
Do you know the value for Light Transmission ?

**Comment by srozum** [comment]
I don't know, here in US 3mm(1/8") and 6mm (1/4") is common thickness of acrylic sheets. As for the transmission, look for 60%.
At the end, you can compensate for different diffuser during calibration by adjusting voltage, unless you don't exceed 12.5V.
